python/mxnet/visualization.py

```diff
@@ -274,11 +278,19 @@ def plot_network(symbol, title="plot", save_format='pdf', shape=None, node_attrs
     draw_shape = False
     if shape is not None:
```

Comment:
not None and if shape aren't the same

in case of 
shape=0
if shape: returns False
if shape is not None: returns True
So, we should be careful about the usage.
Refer - https://stackoverflow.com/questions/7816363/if-a-vs-if-a-is-not-none

Reply:
I agree with Chai - I can see usecase where you want to see shapes but don't have to put any shape information (so `shape={}`), which would return false if changed to `if shape`.